test/test_playbooks/tasks/content_view_filter.yml

```diff
@@ -1,42 +0,0 @@
----
```

Comment:
for those who wonder about this file, it's been unused since that module was introduced in d71ee5eacd3704458aa324ec02d783bfc6fad84e :)